deployment/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,50 @@
+events {}
+http {
+    server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        location / {
+            return 301 https://$host$request_uri;
+        }
+    }
+    server {
+        listen 443 ssl http2;
```

Comment:
http2 is deprecated, see https://forum.hestiacp.com/t/nginx-1-25-1-listen-http2-directive-is-deprecated/9816

Reply:
Also updated that in the next commit